src/dosext/dpmi/dpmi.c

```diff
@@ -380,28 +380,42 @@ static void dpmi_set_pm(int pm)
   dpmi_pm = pm;
 }
 
-int dpmi_is_valid_range(dosaddr_t addr, int len)
+/* gets the most restrictive page attribute (ignoring NX), values are:
```

Comment:
This is a very vague API.
How about verify_read()/verify_write(),
or access_ok() or alike (as in linux kernel)?

Reply:
ok, will do something like that i.e. `dpmi_access_ok(type, addr, size)`

Reply:
I created `dpmi_read_access` and `dpmi_write_access` so it's similar in naming as the vga functions.